compiler-cli/integration-tests/ast/nested_empty_else.mj.stdout

```diff
@@ -0,0 +1,8 @@
+class A {
+	public void f() {
+		if (a)
+			if (b) { }
+			else ;
```

Comment:
>  Falls der else-Teil eines if-Statements ein leeres Statement enthält, so wird der komplette else-Teil
nicht ausgegeben.

```suggestion
```

Reply:
That's exatcly what the test is trying to assert. It should NOT be removed.